config/mssql.yml

```diff
@@ -0,0 +1,7 @@
+doctrine:
+     orm:
+         dql:
+             datetime_functions:
+                 day: DoctrineExtensions\Query\Mysql\Day
```

Comment:
Did you mean `DoctrineExtensions\Query\Mssql\Day`?

Reply:
Good catch 👍